care/users/models.py

```diff
@@ -445,7 +445,7 @@ class UserFacilityAllocation(models.Model):
     facility = models.ForeignKey(
         "facility.Facility", on_delete=models.CASCADE, related_name="+"
     )
-    start_date = models.DateTimeField(default=now)
+    start_date = models.DateTimeField(default=timezone.now)
```

Comment:
left out change, needs to be reverted
```suggestion
    start_date = models.DateTimeField(default=now)
```